File: organization-service/src/main/java/pl/piomin/services/organization/model/Organization.java
package pl.piomin.services.organization.model;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

import java.util.ArrayList;
import java.util.List;

@Getter
@Setter
@ToString
@NoArgsConstructor
public class Organization {

	private Long id;
	private String name;
	private String address;
	private List<Department> departments = new ArrayList<>();
	private List<Employee> employees = new ArrayList<>();
	
	public Organization(String name, String address) {
		this.name = name;
		this.address = address;
	}

}

File: organization-service/src/main/kotlin/pl/piomin/services/organization/model/Organization.kt
package pl.piomin.services.organization.model;

import javax.persistence.*

@Entity
data class Organization(@Id @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "department_id_seq") @SequenceGenerator(name = "department_id_seq", sequenceName = "department_id_seq") var id: Long,
                        var name: String, var address: String) {

    @Transient
    var departments: MutableList<Department> = mutableListOf()
    @Transient
    var employees: MutableList<Employee> = mutableListOf()

}